common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -43,4 +43,14 @@ trait ABTestSwitches {
     sellByDate = Some(LocalDate.of(2022, 1, 10)),
     exposeClientSide = true,
   )
+
+  Switch(
+    ABTests,
+    "ab-integrate-smart",
+    "Integrate Smart AdServer as a Prebid SSP",
+    owners = Seq(Owner.withGithub("frederickobrien")),
```

Comment:
I think it is advisable to make the whole team an owner?

Reply:
Are we able to make the whole team owner here, @mxdvl? It seems `withGithub()` expects a username. Or would "@guardian/commercial-dev" do the trick?

Reply:
I think it would be: `owners = group(Commercial),` as found in the `CommercialSwitches.scala`.